dsl/camel-jbang/camel-jbang-plugin-kubernetes/src/main/java/org/apache/camel/dsl/jbang/core/commands/kubernetes/KubernetesExport.java

```diff
@@ -260,51 +260,30 @@ public Integer export() throws Exception {
 
         if (container.getImagePullPolicy() != null) {
             var imagePullPolicy = container.getImagePullPolicy().getValue();
-            if (runtime == RuntimeType.quarkus) {
-                imagePullPolicy = StringHelper.camelCaseToDash(imagePullPolicy);
-            }
             buildProperties.add("%s.kubernetes.image-pull-policy=%s".formatted(propPrefix, imagePullPolicy));
         }
 
         // Quarkus Runtime specific
         if (runtime == RuntimeType.quarkus) {
 
             // Quarkus specific dependencies
+            // [TODO] migrate to JKube
             if (clusterType != null && clusterType.equals("openshift")) {
                 addDependencies("io.quarkus:quarkus-openshift");
                 if (imageBuilder == null) {
                     // use s2i image builder as a default on OpenShift
                     imageBuilder = "s2i";
                 }
-            } else {
-                addDependencies("io.quarkus:quarkus-kubernetes");
```

Comment:
why did you remove this?

Reply:
because, as we discussed, this is now uniformly handled by jkube

Reply:
Hmm we should surely allow users to use the _quarkus standard_ for running on k8s. And would that not be that you should use their quarkus-kubernetes extension? So you can do all their configurations as well according to their guides.
https://quarkus.io/guides/deploying-to-kubernetes

Reply:
@tdiesler it confuses me that the `io.quarkus:quarkus-openshift` is being added then when cluster-type is "openshift". how is that going to work when we remove all the quarkus settings?

Reply:
@davsclaus the downside of this is that the export for runtime `quarkus` would be different compared to the other runtimes `main` and `spring-boot`. the idea of this PR is to find a common ground for all runtimes so that all exports use pure jKube plugin functionalities. but yes this also means users cannot use the quarkus Kubernetes extension configuration settings OOTB after export @tdiesler I got that right?

Reply:
Yes, the idea was that independent of runtime there is just one way on configuring k8s/openshift resources. The Openshift related code that is currently marked as TODO is untested legacy (which could also be removed in this PR). Support for Openshift will come back with [CAMEL-21013](https://issues.apache.org/jira/browse/CAMEL-21013), also uniformly through JKube.